src/java.security.jgss/share/classes/sun/security/jgss/wrapper/SunNativeProvider.java

```diff
@@ -64,10 +63,10 @@ static void debug(String message) {
         System.err.println(NAME + ": " + message);
     }
 
-    private static final HashMap<String, String> MECH_MAP = constructMechMap();
+    private static final Oid[] MECH_OIDS = constructMechMap();
 
     @SuppressWarnings("restricted")
-    private static HashMap<String, String> constructMechMap() {
+    private static Oid[] constructMechMap() {
```

Comment:
Maybe change the method name? The return value is not a map now.

Reply:
Sounds good. Changed.